dataclients/kubernetes/definitions/routegroupvalidator.go

```diff
@@ -88,6 +99,15 @@ func (rgv *RouteGroupValidator) validatePredicates(item *RouteGroupItem) error {
 				errs = append(errs, err)
 			} else if len(predicates) != 1 {
 				errs = append(errs, fmt.Errorf("%w at %q", errSinglePredicateExpected, p))
+			} else if rgv.EskipValidator != nil {
+				ctx := ResourceContext{
+					ResourceType: "RouteGroup",
```

Comment:
It's not consistent that you chose "ing" for Ingress and here the IMO right way, but documented "ing" vs "rg" :)